platform/avr-ravenusb/cdc_task.c

```diff
@@ -75,6 +75,10 @@
 #include <avr/wdt.h>
 #include <util/delay.h>
 
+#if UIP_CONF_IPV6_RPL
+#include "rpl.h"
+#endif
+
```

Comment:
I added the following comment:
```
// Include needs to be up here instead of embedded in the other
// UIP_CONF_IPV6_RPL block, as doxygen seems not to be compatible with
// #includes embedded in other functions and spits out a warning.
```
The warning:
`contiki/platform/avr-ravenusb/cdc_task.c:582: warning: include file rpl.h not found, perhaps you forgot to add its directory to INCLUDE_PATH?`
As moving the #include to another location fixes the warning, it seems not to be a INCLUDE_PATH problem.